packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts

```diff
@@ -558,4 +558,146 @@ describe('Loan - getLoanInfo', () => {
       })
     }
   })
+
+  it('should count openAuctions', async () => {
+    // extra preps
+    const collateralOracleId2 = await testing.rpc.oracle.appointOracle(
+      await testing.generateAddress(),
+      [
+        // collateral tokens, existed
+        { token: 'DFI', currency: 'USD' },
+        { token: 'BTC', currency: 'USD' },
+        { token: 'ETH', currency: 'USD' },
+
+        // to be loan tokens, haven't exist
+        { token: 'TSLA', currency: 'USD' },
+        { token: 'AMZN', currency: 'USD' }
+      ],
+      { weightage: 1 }
+    )
+    await testing.generate(1)
+    await testing.rpc.oracle.setOracleData(collateralOracleId2, now(), {
+      prices: [
+        { tokenAmount: '1@DFI', currency: 'USD' },
+        { tokenAmount: '10000@BTC', currency: 'USD' },
+        { tokenAmount: '101@ETH', currency: 'USD' },
+        { tokenAmount: '10@TSLA', currency: 'USD' },
+        { tokenAmount: '4@AMZN', currency: 'USD' }
+      ]
+    })
+    await testing.generate(10)
```

Comment:
```suggestion
```
you already have it in `beforeEach`, can use it

Reply:
A new oracle price feeder was required for this test case, therefore added a new one.

Reply:
hmm. why do you need a new oracle? you could update the prices of the existing one?